test/tests/jetty-hello-web/run.sh

```diff
@@ -1,5 +1,7 @@
 #!/bin/bash
 
+[ "$DEBUG" ] && set -x
```

Comment:
Ha! I usually use `git commit --verbose` to see what I'm committing, but it doesn't do any good if you don't actually look at the diff. Let me know if you want me to `--amend` it away.